docs/Code Exposure.md

```diff
@@ -0,0 +1,261 @@
+- CloudWatch and CloudTrail logs stored in S3 can be queried using (Amazon Athena)[https://aws.amazon.com/athena/] to identify undesired actions
+- (Amazon GuardDuty)[https://aws.amazon.com/guardduty/] may automatically detect anomalous traffic.  This can be accessed with (AWS Security Hub)[https://aws.amazon.com/security-hub/] or (Amazon Detective)[https://aws.amazon.com/detective/]
+- Application and instance logs stored in S3 can also be queried using Athena.
+
+### Training
+- `What training is in place for analysts within the company to become familiar with AWS API (command-line environment), Code Commit, S3, RDS, and other AWS services?`
+>>>
+Opportunities here for Threat Detection and incident response include: \
+[AWS RE:INFORCE](https://reinforce.awsevents.com/faq/) \
+[Self-Service Security Assessment](https://aws.amazon.com/blogs/publicsector/assess-your-security-posture-identify-remediate-security-gaps-ransomware/)
+>>>
+
+- `Which roles are able to make changes to services within your account?`
+- `Which users have those roles assigned to them? Is least privilege being followed, or do super admin users exist?`
+- `Has a security assessment been performed against your environment? Do you have a known baseline to detect "new" or "suspicious" things?`
+
+### Communication Technology
+- `What technology is used within the team/company to communicate issues? Is there anything automated?`
+>>>
+Telephone \
+E-mail \
+SMS \
+AWS SES \
+AWS SNS \
+Slack \
+Chime \
+Teams \
+Other? 
+>>>
+
+## Detection
+
+### Logging and S3 Bucket Checks
+- Ensure CloudTrail is enabled in all regions: `./prowler -c check21`
+- Ensure CloudTrail log file validation is enabled: `./prowler -c check22`
+- Ensure S3 bucket access logging is enabled on the CloudTrail S3 bucket: `./prowler -c check26`
+- Ensure there are no S3 buckets open to Everyone or Any AWS user: `./prowler -c extra73`
+- Identify the resources in your organization and accounts; such as Amazon S3 buckets or IAM roles; that are shared with an external entity: `./prowler -c extra769`
+- Find resources exposed to the internet: `./prowler -g group17`
+
+## Escalation Procedures
+- `Who is monitoring the logs/alerts, receiving them and acting upon each?`
+- `Who gets notified when an alert is discovered?`
+- `When do public relations and legal get involved in the process?`
+- `When would you reach out to AWS Support for help?`
+
+## Analysis
+It is highly recommended to export logs to a security incident event management (SIEM) solution (such as Splunk, ELK stack, etc.) to aid in viewing and analyzing a variety of logs for a more complete attack timeline analysis.
+
+### CloudTrail: Public S3 Bucket
+By default, CloudTrail logs API calls that were made in the last 90 days, but not log requests made to objects. You can see bucket-level events on the CloudTrail console. However, you can't view data events (Amazon S3 object-level calls) there—you must parse or query CloudTrail logs for them. 
```

Comment:
"there—you must parse or query CloudTrail logs for them. "

Recommend changing this to "by default--you must enable object level logging before those events appear in CloudTrail."

Reply:
Added recommended change.